samples/qa-a11y/src/views/DatePicker.js

```diff
@@ -0,0 +1,13 @@
+import DatePicker from '@enact/agate/DatePicker';
+import React from 'react';
+
+import Section from '../components/Section';
+
+const DatePickerView = () => (
+	<Section title="Default">
```

Comment:
No aria-labelled section in here?

Reply:
```
<DatePicker alt="Aria-labelled" dayAriaLabel="day" monthAriaLabel="month" yearAriaLabel="year" />
```
Add aria-labelled section with custom aria label props.